app/views/admins/admins/edit.html.slim

```diff
@@ -0,0 +1,38 @@
+.container.p-0
+  .card
+    .card-header
+      h5.card-title.mb-0
+        = t ('admins.passwords.edit.change_password_header')
+    .card-body
+      form
+      = form_for :admin, url: { action: :update } do |form|
+        .row
+          .col-md-12
+              .form-group
+              label
+                = t ('admins.passwords.edit.form.current_password_label')
+              = form.password_field :current_password,
+              class: 'form-control'
+              ol.mb-0.mt-1
+                - @admin.errors[:current_password].each do |msg|
```

Comment:
use this approach 
https://apidock.com/rails/ActionView/Helpers/ActiveRecordHelper/error_message_on

Reply:
and the same for other errors

Reply:
Changed to `errors.full_messages_for`